src/services/compare-appian/serverless.yml

```diff
@@ -0,0 +1,312 @@
+  stackTags:
+    PROJECT: ${self:custom.project}
+    SERVICE: ${self:service}
+  iam:
+    role:
+      path: /delegatedadmin/developer/
+      permissionsBoundary: arn:aws:iam::${aws:accountId}:policy/cms-cloud-admin/developer-boundary-policy
+      statements:
+        - Effect: Allow
+          Action:
+            - dynamodb:GetItem
+            - dynamodb:PutItem
+            - dynamodb:Scan
+          Resource:
+            - ${param:appianTableArn}
+            - ${param:seatoolTableArn}
+            - ${param:statusTableArn}
+        - Effect: Allow
+          Action:
+            - sns:Publish
+          Resource:
+            - ${param:errorsTopicArn}
+        - Effect: Allow
+          Action:
+            - states:StartExecution
+          Resource:
+            - !Ref AppianAlertingStateMachine
+            - !Sub "arn:aws:states:${self:provider.region}:${AWS::AccountId}:execution:${AppianAlertingStateMachine.Name}:*"
+        - Effect: Allow
+          Action:
+            - ses:SendEmail
+            - ses:SendRawEmail
+          Resource:
+            - !Sub "arn:aws:ses:${self:provider.region}:${AWS::AccountId}:identity/*"
+        - Effect: Allow
+          Action:
+            - secretsmanager:GetSecretValue
+          Resource:
+            - !Sub "arn:aws:secretsmanager:${self:provider.region}:${AWS::AccountId}:secret:${self:custom.project}/${sls:stage}/alerts*"
+        - Effect: Allow
+          Action:
+            - secretsmanager:ListSecrets
+          Resource: "*"
+        - Effect: Allow
+          Action:
+            - logs:PutLogEvents
+          Resource:
+            - ${param:sesLogGroupArn}
+
+params:
+  master:
+    initialWaitSec: 86400 # 1 days
+    tempWaitSec: 172800 # 2 days
+    sinceSubmissionChoiceSec: 345600 # 4 days
+    recordAgeChoiceSec: 17366000 # 201 days
+  val:
+    initialWaitSec: 86400
+    tempWaitSec: 172800
+    sinceSubmissionChoiceSec: 345600
+    recordAgeChoiceSec: 17366000
+  production:
+    initialWaitSec: 86400
+    tempWaitSec: 172800
+    sinceSubmissionChoiceSec: 345600
+    recordAgeChoiceSec: 17366000
+  default:
+    initialWaitSec: 2
+    tempWaitSec: 43200 # 12 hours
+    sinceSubmissionChoiceSec: 3
+    recordAgeChoiceSec: 17366000 # 201 days
+
+custom:
+  project: ${env:PROJECT}
+  serverlessTerminationProtection:
+    stages: # Apply CloudFormation termination protection for these stages
+      - master
+      - val
+      - production
+  secretId: ${self:custom.project}/${sls:stage}/alerts
+  scripts:
+  commands:
+    stop-executions: |
+      aws stepfunctions list-executions \
+      --state-machine-arn arn:aws:states:${self:provider.region}:${aws:accountId}:stateMachine:${self:service}-${sls:stage}-compare-alerting \
+      --status-filter RUNNING \
+      --query "executions[*].{executionArn:executionArn}" \
+      --output text | \
+      xargs -I {} aws stepfunctions stop-execution \
+      --execution-arn {} >> out.t
+      rm out.t
```

Comment:
i found out this script has issues, so we opted for a different pattern. we are using a custom resource instead, if you follow this change: https://github.com/CMSgov/seatool-compare/pull/88/files i think the only thing that would be different is the `stateMachineArn: !Ref MMDLAlertingStateMachine`